### ISSUE TITLE
feat: ✨ provide option to pass aria-label to underlaying component (sd-button, sd-link)

### ISSUE THREAD

**Issue by christophsaile** [solid-design-system/solid#1106]
## User Story
As a User of the Solid Design System, I would like to set an aria-label for the sd-button and sd-link component, in order to define a custom text for AT users.

### Suggested Solution
Add a prop which makes it possible to pass an aria-lable to the underlaying `<button>`, `<a> `element of the `<sd-button>`  and `<sd-link>` component.
If you currently add an aria-label to the `<sd-button>` (with a href set) the `<a>` does not have the correct text in the VO rotor see screenshot.
![filename](https://github.com/solid-design-system/solid/assets/54807480/27f72352-4769-4a0b-8f12-ba3a6477fb2a)

Furthermore VO will read out the aria-label and the link content and group it:
![image](https://github.com/solid-design-system/solid/assets/54807480/c3790950-9f04-4225-a941-3fc835598fa7)

The following element was used for testing -> top left one:
![image](https://github.com/solid-design-system/solid/assets/54807480/a7417702-be52-4c42-9937-3763e077428b)

![image](https://github.com/solid-design-system/solid/assets/54807480/47ecca8e-30c2-43e5-b843-8b3904edf3b4)

## Subtasks
- [ ] Design_implementation_task
- [ ] Development_implementation_task
- [ ] Design_release_task

## DoR
- [ ] Item has business value
- [ ] Item all subtasks have been estimated by the team
- [ ] Item is clear and well-defined
- [ ] Item dependencies have been identified

## DoD
- [ ] All subtasks have been closed

**Comment by christophsaile** [comment]
As @mariohamann suggested, writing the markup like this fixes the issue:

```html
<sd-button href="www.ecosia.org">
  <span aria-hidden="true">Read more</span>
  <span class="visually-hidden sr-only">This is the content for screenreaders</span>
</sd-button>
```

![image](https://github.com/solid-design-system/solid/assets/54807480/72d57466-d7c3-46c4-8fb4-439876547c11)

**Comment by christophsaile** [comment]
I updated the ticket

**Comment by mariohamann** [comment]
Connected to https://github.com/solid-design-system/solid/issues/1926
